hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -317,6 +320,7 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Request, log logr.Logger, hcluster *hyperv1.HostedCluster) (ctrl.Result, error) {
 	controlPlaneNamespace := manifests.HostedControlPlaneNamespace(hcluster.Namespace, hcluster.Name)
 	hcp := controlplaneoperator.HostedControlPlane(controlPlaneNamespace.Name, hcluster.Name)
+	var hcDestroyGracePeriod time.Duration
```

Comment:
this is just between the declaration of hcp and its usage. Please let's declare hcDestroyGracePeriod in line 403 along with its usage